client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -17,14 +17,17 @@ import {
  * Internal dependencies
  */
 import { getContactDetailsExtraCache } from 'state/selectors';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
 import { updateContactDetailsCache } from 'state/domains/management/actions';
 import FormFieldset from 'components/forms/form-fieldset';
 import FormLabel from 'components/forms/form-label';
 import FormSelect from 'components/forms/form-select';
 import FormCheckbox from 'components/forms/form-checkbox';
+import FormInputValidation from 'components/forms/form-input-validation';
 
 const ciraAgreementUrl = 'https://services.cira.ca/agree/agreement/agreementVersion2.0.jsp';
 const defaultValues = {
+	lang: 'EN',
 	legalType: 'CCT',
 	ciraAgreementAccepted: false,
```

Comment:
So, I flipped this back `false` because the forced agree felt weird to me. I ended up running it past a8c legal as I started waffling and they do prefer it unchecked.